webapp/assert/index.html

```diff
@@ -64,70 +64,73 @@
 
   <main class="container-fluid">
-    <div id="pageContent" class="container-xxl py-4">
-      <h1 class="pt-5 text-center">Hello!</h1>
-      <div class="fs-3">
-        <p>
-          It has been nearly 6 months since the computershared.net reddit scraper has seen a post from you. This could
-          be because it's goofed up, or something else.
-        </p>
-        <p>
-          You have a few options. Would you like to:
-        <ul>
-          <li class="pb-4">Assert that the most recent post still accurately reflects your account balance
-            </br>
-            <button class="btn btn-secondary">Assert</button>
-          </li>
-          <li class="pb-4">Let the scraper know that it missed a more recent post
-            </br>
-            <button class="btn btn-secondary">Missed one</button>
-          </li>
-          <li class="pb-4">Make a new post on Reddit with an up-to-date screenshot
-            </br>
-            <a href="https://www.reddit.com/">
-              <button class="btn btn-secondary">Take me to Reddit</button>
-            </a>
-          </li>
-          <li class="pb-4">Let the reddit scraper know that you've closed or lowered your position:
-            </br>
-            <div class="input-group">
-              <input type="text" class="form-control text-end" placeholder="0" style="max-width: 10em;" />
-              <button class="btn btn-secondary">Update my Balance</button>
-            </div>
-          </li>
-          <li class="pb-4">Do nothing at all. I don't mind being dropped from computershared.net's sample set
-            </br>
-            <a href="../">
-              <button class="btn btn-secondary">Wave goodbye</button>
-            </a>
-          </li>
-        </ul>
-        Whatever you choose, thank you for taking the time to help computershared.net.
-        </p>
+    <div id="pageContent" class="container-xxl">
+      <div class="row">
+        <div class="col-12 fs-4">
+          <h1>Confirm your Share Data!</h1>
+          <h2 class="pt-5"><i class="fab fa-reddit-alien"></i> Hello Reddit Person!</h2>
+          <div class="fs-4  body-copy">
+            <p>It has been nearly 6 months since the computershared.net reddit scraper has seen a post from you, which
+              means
+              your shares are soon going to <a href="#" target="_blank">drop out</a> of the data we show on this site.
+            </p>
+            <p>This could be because the scraper goofed up, or perhaps you havent added any shares for a while or just
+              didnt
+              post on Reddit. If you still want us to count your shares, you have a few easy options:</p>
+            <ul class="assert-options">
+              <li class="pb-4"><span>Assert that your most <a href="#" target="_blank">recent post</a> still accurately
+                  reflects your Computershare GME account balance</span>
+                <button class="btn btn-primary">Yep, it's correct</button>
```

Comment:
The primary color (blue) is gross and doesn't jive with the theme. Perhaps a derivative of computershare purple?